### ISSUE TITLE
Issue with "c-review-type" on crosswalk (problem on Elements' end)

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#289]
Hi all (especially @amandafrench ), I just wanted to note that Elements has an unwieldy way of exporting metadata from the "c-review-type" field ("Peer reviewed?"), which is an option that users can select when uploading conference publications. I don't think that there's anything we can do on our end about this issue except remove the "c-review-type" field altogether from the mapping, but I just wanted to bring it to your attention. I also want to emphasize that I don't think that irregularities in this one field - which we likely can't fix (or would have to remove) - should hold up the deployment of our Irish Setter changes. 

To elaborate, users who submit conference items in Elements are given the option of checking one or more options for the "c-review-type" field, which translates to "Peer reviewed?" (see screenshot below).

![screen shot 2016-06-10 at 7 04 06 pm](https://cloud.githubusercontent.com/assets/5408616/15981013/21ca2e68-2f3e-11e6-8900-346eff8969f5.png)

It is possible for users to select more than option in this field. If a user does this, the following information appears in the Elements' record:

![screen shot 2016-06-10 at 7 05 19 pm](https://cloud.githubusercontent.com/assets/5408616/15981036/676056a0-2f3e-11e6-938f-6646d1acc112.png)

As a result, the following information is exported to VTechWorks (commas don't appear, for whatever reason):

![screen shot 2016-06-10 at 7 07 36 pm](https://cloud.githubusercontent.com/assets/5408616/15981068/b85c0df6-2f3e-11e6-9cae-64c2e101137b.png)

My point is that this information is strung together and looks quite messy. As far as I know, I can't change the way this information is displayed because ALL of the metadata from the "Peer reviewed?" field is mapped to "c-review-type" - I can't split up "Yes, full paper", "Yes, abstract only", and "No". Also, this issue happens ONLY if a user selects all 3 options.

So, in a nutshell, I'll open a support ticket on Symplectic's site and put in a feature request, but I don't think that this is something that we can fix (unless we remove the "c-review-type" mapping altogether), and I do NOT think that this should hold up the Irish Setter deployment. Because everything else seems to be working (knock on wood), I would vote for addressing this in the next sprint. Thanks so much, and take care.

**Comment by mello99** [comment]
Just put in a support ticket on Symplectic's site - https://support.symplectic.co.uk/support/tickets/21301. Also, here's a link to the feature request I put on Symplectic's site as well - https://support.symplectic.co.uk/support/discussions/topics/6000038020/page/last#post-6000081466

**Comment by mello99** [comment]
Hi @amandafrench and @alawvt  (I know you're reading this, Anne, lol!), after some thought, I would stash this issue under "much ado about nothing", but will leave it up to @amandafrench to close it. 

In essence, if a user selects all 3 options, they've misunderstood the question (or done so accidentally). If a user selects both "Yes, full paper" and "Yes, abstract only", they've made a logical error since the full paper would include the abstract, thus negating the need to select both options. Perhaps this isn't phrased very clearly, but I think the question's intent is clear. Selecting "Yes, full paper" or "Yes, abstract only" AND "No" is an obvious logical error. 

I'll put in a feature request on Symplectic's support site and ask that they only allow users to select one answer for this field. Thanks!

**Comment by alawvt** [comment]
@mello99, I think that requesting that users only be allowed to select one option makes sense. Thanks for reporting these issues with very clear detail.

**Comment by mello99** [comment]
Ok, Symplectic support gave me some code to work with, plus I put in a [feature request](https://support.symplectic.co.uk/support/discussions/topics/6000038020), so I'm closing this issue. Thanks, all!
